<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportedBookmarksPreviewDialog</class>
 <widget class="QDialog" name="ImportedBookmarksPreviewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Bookmarks</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
   <item row="0" column="0">
    <widget class="QWidget" name="widLeftPane" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel { font-weight: bold; }</string>
        </property>
        <property name="text">
         <string>The following bookmarks were imported. Folder hierarchies will not be preserved, instead tags will be used. You can choose tag to apply to all or specific bookmarks.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Apply these tags to all imported entries:</string>
        </property>
        <property name="buddy">
         <cstring>leTagsForAll</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TagLineEdit" name="leTagsForAll"/>
      </item>
      <item>
       <widget class="QCheckBox" name="chkRemoveImportedFile">
        <property name="text">
         <string>Remove files from system after import</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>38</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="grpDuplBookmarkProps">
        <property name="styleSheet">
         <string notr="true">QGroupBox { font-weight: bold; }</string>
        </property>
        <property name="title">
         <string>Duplicate Bookmark Properties</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Another bookmark with this url and &lt;span style=&quot; font-style:italic;&quot;&gt;almost similar&lt;/span&gt; details already exists in the database. What should be done?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>optKeep</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QRadioButton" name="optKeep">
           <property name="styleSheet">
            <string notr="true">QRadioButton { font-weight: bold; }</string>
           </property>
           <property name="text">
            <string>Keep existing bookmark</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Don't import the new bookmark. &lt;span style=&quot; font-weight:600; color:#800000;&quot;&gt;Warning: &lt;/span&gt;Any details that the new bookmark has is discarded!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QRadioButton" name="optOverwrite">
           <property name="styleSheet">
            <string notr="true">QRadioButton { font-weight: bold; }</string>
           </property>
           <property name="text">
            <string>Import and overwrite with the new bookmark</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#FF0000;&quot;&gt;Careful: &lt;/span&gt;All attached files, tags, linked bookmarks and other additional information &lt;span style=&quot; font-weight:600;&quot;&gt;will be removed&lt;/span&gt; from the previous bookmark without an undo option.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QRadioButton" name="optAppend">
           <property name="styleSheet">
            <string notr="true">QRadioButton { font-weight: bold; }</string>
           </property>
           <property name="text">
            <string>Keep details for both</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#008000;&quot;&gt;Safest Option. &lt;/span&gt;Adds any additional new detail to the existing bookmark.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpDuplBookmarkSameProps">
        <property name="styleSheet">
         <string notr="true">QGroupBox { font-weight: bold; }</string>
        </property>
        <property name="title">
         <string>Duplicate Bookmark With Same Properties</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Another bookmark with this url already exists in the database, but the details of the bookmarks are the same. The new bookmark will not be imported.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpBookmarkProps">
        <property name="styleSheet">
         <string notr="true">QGroupBox { font-weight: bold; }</string>
        </property>
        <property name="title">
         <string>Bookmark Properties</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="chkImportBookmark">
           <property name="text">
            <string>&amp;Import this bookmark</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Apply these additional tags to this bookmark:</string>
           </property>
           <property name="buddy">
            <cstring>leTagsForBookmark</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TagLineEdit" name="leTagsForBookmark"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpBookmarkFolderProps">
        <property name="styleSheet">
         <string notr="true">QGroupBox { font-weight: bold; }</string>
        </property>
        <property name="title">
         <string>Bookmark Folder Properties</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="chkImportFolder">
           <property name="text">
            <string>&amp;Import bookmarks in this folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Apply these additional tags to all bookmarks in this folder:</string>
           </property>
           <property name="buddy">
            <cstring>leTagsForFolder</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="TagLineEdit" name="leTagsForFolder"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTreeWidget" name="twBookmarks">
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Tags/TagLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leTagsForAll</tabstop>
  <tabstop>chkRemoveImportedFile</tabstop>
  <tabstop>optKeep</tabstop>
  <tabstop>optOverwrite</tabstop>
  <tabstop>optAppend</tabstop>
  <tabstop>chkImportBookmark</tabstop>
  <tabstop>leTagsForBookmark</tabstop>
  <tabstop>chkImportFolder</tabstop>
  <tabstop>leTagsForFolder</tabstop>
  <tabstop>twBookmarks</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportedBookmarksPreviewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportedBookmarksPreviewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
